src/pages/TransactionDetailsPage.tsx

```diff
@@ -332,6 +332,15 @@ function StatusCard({
           </tr>
         )}
 
+        {computeUnitsConsumed && (
+          <tr>
+            <td>Compute units consumed</td>
+            <td className="text-lg-end">
+              {computeUnitsConsumed.toLocaleString("en-US")}
+            </td>
+          </tr>
+        )}
```

Comment:
For a system instruction, in which this is `0`, you get a funny result :sweat_smile: can you do?
```suggestion
        {computeUnitsConsumed !== undefined && (
          <tr>
            <td>Compute units consumed</td>
            <td className="text-lg-end">
              {computeUnitsConsumed.toLocaleString("en-US")}
            </td>
          </tr>
        )}
```

Reply:
![image](https://user-images.githubusercontent.com/934662/230168789-2250a414-4b53-4f76-b0bf-583eb9870ef6.png)

Reply:
The classic react noob bug